WalletWasabi.Backend/Config.cs

```diff
@@ -175,41 +175,21 @@ public async Task<bool> CheckFileChangeAsync()
 			string jsonString = await File.ReadAllTextAsync(FilePath, Encoding.UTF8);
 			var config = JsonConvert.DeserializeObject<Config>(jsonString);
 
-			if (Network != config.Network)
-			{
-				return true;
-			}
-			if (BitcoinRpcConnectionString != config.BitcoinRpcConnectionString)
+			if (Network == config.Network
```

Comment:
Could inline this as a return

Reply:
That is true, however it will be very difficult to read and to maintain.
IMO conditional expression should be used only when the condition is simple.

Reply:
I disagree, but we can see what other say